ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c

```diff
@@ -795,8 +795,6 @@ input_listener_input_acceptance(void *data,
     }
 
     if (!surface_found) {
-        fprintf(stderr, "Warning: input acceptance event received for "
-                "nonexistent surface %d\n", surface);
```

Comment:
We should not send any input events for background surface. ivi-input-controller can also check for bkgnd surface id and not send any acceptance event. It would be even easier when PR of Michael is merged. Then, ivi-input-controller can get the bkgnd surface id directly from ivishell struct.